tokio/src/sync/broadcast.rs

```diff
@@ -957,7 +957,6 @@ where
 
 #[cfg(feature = "stream")]
 #[doc(hidden)]
-#[deprecated(since = "0.2.21", note = "use `into_stream()`")]
```

Comment:
FWIW, I don't think I would have known (e.g. `E-easy`) that its ok to just drop this deprecated attribute on this branch!

Reply:
Luckily people are allowed to ask questions.

Reply:
Many, perhaps wiser than me, might not bother, expecting they won't get a response to inconvenient questions. :slightly_smiling_face: